src/main/java/com/google/firebase/remoteconfig/internal/TemplateResponse.java

```diff
@@ -110,27 +96,22 @@ public RemoteConfigParameter toRemoteConfigParameter() {
     @Key("useInAppDefault")
     private Boolean inAppDefaultValue;
 
-    public ParameterValueResponse() {
-    }
-
-    private ParameterValueResponse(String value, Boolean inAppDefaultValue) {
-      this.value = value;
-      this.inAppDefaultValue = inAppDefaultValue;
+    public String getValue() {
+      return value;
     }
 
-    public static ParameterValueResponse ofValue(String value) {
-      return new ParameterValueResponse(value, null);
+    public Boolean isInAppDefaultValue() {
```

Comment:
`is***()` methods should never return null. Instead change this to one of the following:

```java
public Boolean getUseInAppDefault() {
  return this.useInAppDefault;
}

public boolean isUseInAppDefault() {
  return Boolean.TRUE.equals(this.useInAppDefault);
}
```

Reply:
Ah good point! Thanks!